testing/integration/install_unix_test.go

```diff
@@ -17,11 +18,12 @@ import (
 
 	"github.com/elastic/elastic-agent/internal/pkg/agent/application/paths"
 	"github.com/elastic/elastic-agent/internal/pkg/agent/install"
+	atesting "github.com/elastic/elastic-agent/pkg/testing"
 )
 
-func checkPlatformUnprivileged(t *testing.T, topPath string) {
+func checkPlatformUnprivileged(t *testing.T, _ *atesting.Fixture, topPath string) {
 	// Check that the elastic-agent user/group exist.
-	uid, err := install.FindUID("elastic-agent")
+	uid, err := install.FindUID("elastic-agent-user")
 	require.NoError(t, err)
 	gid, err := install.FindGID("elastic-agent")
```

Comment:
Nit: maybe we should use the constants for user and group here, just to be on the safe side

Reply:
Switched to constants.